src/pages/_app.tsx

```diff
@@ -1,23 +1,25 @@
 import "@/styles/globals.css";
 import type { AppProps } from "next/app";
 import { Inter, Staatliches } from "@next/font/google";
-import { QueryClient, useQuery } from "@tanstack/react-query";
+import {
+	QueryClient,
+	QueryClientProvider,
+	useQuery,
+} from "@tanstack/react-query";
 import { useState } from "react";
-import { ReactQueryDevtools } from "@tanstack/react-query-devtools";
 import { SessionProvider } from "next-auth/react";
-import { QueryClientProvider } from "@tanstack/react-query";
 const queryClient = new QueryClient();
 
 export default function App({ Component, pageProps }: AppProps) {
-  const [queryClient] = useState(() => new QueryClient());
+	const [queryClient] = useState(() => new QueryClient());
 
-  return (
-    <div className="bg-grad-frame h-screen w-full px-8">
-      <QueryClientProvider client={queryClient}>
-        <SessionProvider>
-          <Component {...pageProps} />
-        </SessionProvider>
-      </QueryClientProvider>
-    </div>
-  );
+	return (
+		<div className="bg-grad-frame h-screen w-auto">
```

Comment:
are you sure? The app should take the full width of the viewport.
This will be overwritten by changes that @amyria3 is making, you can just remove these changes from this PR

Reply:
changed